x/dex/keeper/pair_helper.go

```diff
@@ -25,3 +27,7 @@ func (k Keeper) SortTokens(ctx sdk.Context, tokenA string, tokenB string) (strin
 func (k Keeper) CreatePairId(token0 string, token1 string) (pairId string) {
 	return (token0 + "<>" + token1)
 }
+
+func (k Keeper) CreateSharesId(token0 string, token1 string, tickIndex int64, fee int64) (denom string) {
+	return fmt.Sprintf("%s%st%df%d", token0, token1, tickIndex, fee)
```

Comment:
is this vulnerable to injection? probably should sanitize these inputs and use delimiters

Reply:
fixed to TokenA-TokenB-t0-f0